package.json

```diff
@@ -133,16 +143,20 @@
     "webpack-cli": "^4.6.0"
   },
   "scripts": {
-    "start": "RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy yarn _private:start",
-    "start:dev": "RADICLE_UPSTREAM_PROXY_PATH=./target/debug/radicle-proxy yarn _private:start:dev",
-    "start:test": "RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy RADICLE_UPSTREAM_PROXY_ARGS='--test' yarn _private:start",
+    "start": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy yarn _private:start",
+    "start:win": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/release/radicle-proxy.exe yarn _private:start",
+    "start:dev": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/debug/radicle-proxy yarn _private:start:dev",
+    "start:dev:win": "cross-env RADICLE_UPSTREAM_PROXY_PATH=./target/debug/radicle-proxy.exe yarn _private:start:dev",
```

Comment:
I’m wondering whether the proxy path should be detected by the app. We would only need one script then.

Reply:
Yes, I've considered this also, but it was more effort

Reply:
:ok_hand: Would you mind creating an issue for this after the PR is merged?

Reply:
Done in https://github.com/radicle-dev/radicle-upstream/pull/2375.